src/index.js

```diff
@@ -9,25 +9,16 @@ import Token from './resources/token'
 import fetch from 'cross-fetch'
 import store from 'store'
 
-const defaultBridgeServer = 'https://bridge.originprotocol.com'
-const defaultIpfsDomain = 'gateway.originprotocol.com'
-const defaultDiscoveryServer = 'https://discovery.originprotocol.com'
-const defaultDiscoveryServerPort = '443'
-const defaultIpfsApiPort = '5002'
-const defaultIpfsGatewayPort = '443'
-const defaultIpfsGatewayProtocol = 'https'
-const defaultAttestationServerUrl = `${defaultBridgeServer}/api/attestations`
 const VERSION = require('.././package.json').version
 
 class Origin {
   constructor({
-    ipfsDomain = defaultIpfsDomain,
```

Comment:
I believe this will break our production deployment when the DApp is served via IPFS because in that case no environment variable is defined. So we must keep those default assignments...